Makefile

```diff
@@ -110,16 +110,19 @@ tags: http_parser.c http_parser.h test.c
 	ctags $^
 
 install: library
-	install -D $(SONAME) $(LIBDIR)/$(SONAME)
-	-mv http_parser.h $(INCLUDEDIR)/http_parser.h
- 
+        install -D $(SONAME) $(LIBDIR)/$(SONAME)
+        ln -s $(LIBDIR)/$(SONAME) $(LIBDIR)/libhttp_parser.so
+        install -D  http_parser.h $(INCLUDEDIR)/http_parser.h
+
 install-strip: library
-	install -D -s $(SONAME) $(LIBDIR)/$(SONAME)
-	-mv http_parser.h $(INCLUDEDIR)/http_parser.h
- 
+        install -D -s $(SONAME) $(LIBDIR)/$(SONAME)
+        ln -s $(LIBDIR)/$(SONAME) $(LIBDIR)/libhttp_parser.so
```

Comment:
This is incredibly wrong and utterly violates the fundamental aspect of DESTDIR. See https://www.gnu.org/prep/standards/html_node/DESTDIR.html

Also see #264 #272 #321 and #346 which all try to fix this and thereby allow the Makefile to actually work with DESTDIR, all of which are busy being ignored.